platform/core/src/utils/metadataProvider/getPixelSpacingInformation.js

```diff
@@ -67,24 +75,56 @@ export default function getPixelSpacingInformation(instance) {
       type: TYPES.CALIBRATED,
       isProjection,
       PixelSpacingCalibrationType,
-      PixelSpacingCalibrationDescription
+      PixelSpacingCalibrationDescription,
     };
   } else if (!PixelSpacing && ImagerPixelSpacing) {
     let CorrectedImagerPixelSpacing = ImagerPixelSpacing;
     if (EstimatedRadiographicMagnificationFactor) {
       // Note that in IHE Mammo profile compliant displays, the value of Imager Pixel Spacing is required to be corrected by
       // Estimated Radiographic Magnification Factor and the user informed of that.
       // TODO: should this correction be done before all of this logic?
-      CorrectedImagerPixelSpacing = ImagerPixelSpacing.map(pixelSpacing => pixelSpacing / EstimatedRadiographicMagnificationFactor);
+      CorrectedImagerPixelSpacing = ImagerPixelSpacing.map(
+        pixelSpacing => pixelSpacing / EstimatedRadiographicMagnificationFactor
+      );
     } else {
-      log.info('EstimatedRadiographicMagnificationFactor was not present. Unable to correct ImagerPixelSpacing.');
+      log.info(
+        'EstimatedRadiographicMagnificationFactor was not present. Unable to correct ImagerPixelSpacing.'
+      );
     }
 
     return {
       PixelSpacing: CorrectedImagerPixelSpacing,
       isProjection,
     };
+  } else if (
+    SequenceOfUltrasoundRegions &&
```

Comment:
It might be a bit cleaner to check if `SequenceOfUltrasoundRegions` exists, then do a nested if which just checks Array.isArray, and else. Rather than two `else if`s that check the same thing at the start of their conditional.